src/dcos_e2e/_common.py

```diff
@@ -7,8 +7,44 @@
 from subprocess import PIPE, STDOUT, CompletedProcess, Popen
 from typing import Dict, List, Optional, Union
 
-logging.basicConfig(level=logging.DEBUG)
-LOGGER = logging.getLogger(__name__)
+
+def get_logger(name: str) -> logging.Logger:
+    """
+    Return a customized logger that is able to log on DEBUG level.
+
+    Args:
+        name: Name of the logger. Setting this to ``__name__`` will log
+            the path to the file where the logger has been created.
+
+    Returns:
+        See :py:class:`logging.Logger`.
+    """
+    # This gets the root logger with a ``logging.lastResort``
+    # StreamHandler that logs on WARNING level.
+    logger = logging.getLogger(name)
+
+    # Do this if a logger is called for the first time
+    if not logger.hasHandlers():
```

Comment:
Will this get 100% coverage? That is, does the test suite cover both `logger.hasHandlers() is True` and `logger.hasHandlers() if False`?